packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -109,7 +117,8 @@ contract Reserve is
   function initialize(
```

Comment:
No need to modify the initialize in my perspective, but you could change the names of the arguments to reflect that number is meant to be Celo

Reply:
done